### DISCUSSION TITLE
How to use Stereokit to develop overlay applications？How Stereokit interacts with OpenXL？

### DISCUSSION THREAD

**Discussion by jike2018** [StereoKit/StereoKit#908] [Q&A]
## Describe the feature
  I want to overlay a portion of the image rendered by Stereokit onto other Android applications, just like overlay. May I ask what should be done? Or what materials should I look at?

Similar to how Android system prompts overlay on other applications.

**Comment by maluoi** [comment]
The first requirement is that you're running on an OpenXR runtime that supports the `XR_EXTX_overlay` extension! Right now that's Monado, and Monado based runtimes like Snapdragon Spaces.

After that, all you need to do from StereoKit's side is mark it as an overlay app during initialization, with [`SKSettings.overlayApp`](https://stereokit.net/Pages/StereoKit/SKSettings/overlayApp.html), maybe play with the order priority with `SKSettings.overlayPriority`. That should be enough? It's possible you may want to make your overlay app a foreground `Service` rather than an `Activity`, all the overlay apps I've written have ended up as some kind of `Service` like that.

**Comment by FanEric** [comment]
Hi, @jike2018, have you achieved it?
